local_store.go

```diff
@@ -82,12 +94,13 @@ func (m *memoryStore) Commit(consistentSnapshot bool, versions map[string]int, h
 	return nil
 }
 
-func (m *memoryStore) GetSigningKeys(role string) ([]sign.Signer, error) {
+func (m *memoryStore) GetSigners(role string) ([]keys.Signer, error) {
 	return m.signers[role], nil
 }
 
-func (m *memoryStore) SavePrivateKey(role string, key *sign.PrivateKey) error {
-	m.signers[role] = append(m.signers[role], key.Signer())
+func (m *memoryStore) SavePrivateKey(role string, key *data.PrivateKey) error {
```

Comment:
might just be better to make it take in the Signer directly and call it `SaveSigner`.

I don't like `Create` -- because the signer was already created, just represented as the data.PrivateKey.

Reply:
Renamed to SaveSigner and take in a signer